src/commands/warn.ts

```diff
@@ -0,0 +1,78 @@
+import { CommandInteraction } from "discord.js";
+import { SlashCommandBuilder, userMention, hideLinkEmbed } from "@discordjs/builders";
+import { getTexts } from "../common";
+
+const CONDUCT = "conduct";
+const CONTENT = "content";
+const SPAM = "spam";
+type Reason = typeof CONDUCT | typeof CONTENT | typeof SPAM;
```

Comment:
`Reason` type shouldn't be necessary. `as Reason` below doesn't do anything.  
Just inline them, e.g., `name: "conduct"`.